Tools/TestWebKitAPI/Tests/WebKitGLib/TestWebKitWebContext.cpp

```diff
@@ -378,7 +378,7 @@ static void testWebContextURIScheme(URISchemeTest* test, gconstpointer)
 
     // Torture test time: make sure it still works if we issue a bunch of different requests all at
     // once. Each request should finish and return exactly the same data.
-    int numIterations = 25;
+    const int numIterations = 25;
```

Comment:
Why not `static constexpr` to guarantee compile-time evaluation?

```suggestion
    static constexpr int numIterations = 25;
```

Reply:
Sure, whatever.